ci-operator/jobs/openshift/hypershift/openshift-hypershift-release-4.16-periodics.yaml

```diff
@@ -716,14 +716,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-kubevirt-metal
-  reporter_config:
-    slack:
-      channel: '#hypershift-kubevirt-platform'
-      job_states_to_report:
-      - failure
-      report_template: ':volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs> :volcano:'
```

Comment:
we need the slack reporting to remain. it gets removed when the job name is changed, then we have to add it back

Reply:
I didn't delete the report
https://github.com/openshift/release/pull/52997/files#diff-ea96f751fa83f7954b399383682b4c97d9c8006d024da8e737e17c9fa58981feR800-R807